_includes/v19.1/known-limitations/homebrew.md

```diff
@@ -0,0 +1,2 @@
+<p>It is not currently possible to install CockroachDB v19.1 via Homebrew due to an incompatibility between CockroachDB and Go 1.12.1, which Homebrew currently uses. Once Go 1.12.2 is released, this incompatibility will be resolved, and installing this version via Homebrew will be possible again.
```

Comment:
"on certain versions of macOS" after "Go 1.12.1".

Reply:
Done.